src/main/java/com/pubnub/api/managers/ReconnectionManager.java

```diff
@@ -106,7 +106,7 @@ private void stopHeartbeatTimer() {
 
     private void callTime() {
         try {
-            pubnub.time().async(new PNCallback<PNTimeResult>() {
+            pubnub.time().async(new PNResultCallback<PNTimeResult>() {
```

Comment:
We could theoretically convert these to lambdas now, but I'm trying to keep the diff small